cmd/minishift/cmd/config/config.go

```diff
@@ -44,6 +43,22 @@ type Setting struct {
 	callbacks   []setFn
 }
 
+type Flags struct {
+	Name        string
+	set         func(MinishiftConfig, string, string) error
+	validations []setFn
+	callbacks   []setFn
+}
+
+var (
+	SetUrl = Flags{
+		Name:        "iso-url",
+		set:         SetString,
+		validations: []setFn{IsValidUrl},
+		callbacks:   []setFn{RequiresRestartMsg},
+	}
+)
+
```

Comment:
@hferentschik FYI, this is how each flag will look when we have it in one place. Hope this is what we discussed. Currently I didn't put those options to a different file. Let me know if you have something else in mind.